crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__test__schema_sdl_export.snap

```diff
@@ -61,7 +111,7 @@ type Object implements ObjectOwner {
 	coinConnection(first: Int, after: String, last: Int, before: String, type: String): CoinConnection
 	stakeConnection(first: Int, after: String, last: Int, before: String): StakeConnection
 	defaultNameServiceName: String
-	nameServiceConnection(first: Int, after: String, last: Int, before: String): NameServiceConnection
+	nameServiceConnection(first: Int, after: String, last: Int, before: String): StringConnection
```

Comment:
Only issue is this -- can we create a wrapper type to make this retain its name?